docs/source/cli/model-training-inference/real-time-inference-spec.rst

```diff
@@ -0,0 +1,229 @@
+..........................................................
+
+Depending on the value of ``gml_task``, a target object in a ``targets`` array could be a ``node`` object
+or an ``edge`` object as defined above. As a target object, the ``features`` field is not required. 
+
+    .. note::
+
+        A target objects, a ``node`` or an ``edge``, should have the same ``node`` or ``edge`` object
+        in the ``nodes`` or ``edges`` array. For example, in the below payload example, the ``author``
+        node ``a39`` is a target node, and it also is one of the nodes in the ``nodes`` list.
+
+        .. code:: yaml
+
+            {
+                "version": "gs-realtime-v0.1",
+                "gml_task": "node_classification",
+                "graph": {
+                    "nodes": [
+                        {
+                            "node_type": "author",
+                            "node_id": "a4444",
+                            "features": { ...... },
+                        },
+                        {
+                            "node_type": "author",
+                            "node_id": "a39",
+                            "features": { ...... },
+                        },
+                        ......
+                    ],
+                    "edges": [ ......]
+                },
+                "targets": [
+                    {
+                        "node_type": "author",
+                        "node_id": "a39"
+                    }
+                ]
+            }
+
+.. _rt-response-body-spec:
+
+Specification of Response Body Contents 
+----------------------------------------
+
+A response body is a JSON object.
+
+**Response Body Syntax**:
+.........................
+
+.. code:: json
+
+    {
+        'status_code'   : 'int',
+        'request_uid'   : 'string',
+        'message'       : 'string',
+        'error'         : 'string',
+        'data'          : {
+            results: [
+                {
+                    'node_type'     : 'string',
+                    'node_id'       : 'string',
+                    'predictions'   : [ ...... ]
+                },
+                or
+                {
+                    'edge_type'     : ['string', 'string', 'string'],
+                    'src_node_id'   : 'string',
+                    'dest_node_id'  : 'string',
+                    'predictions'   : [ ...... ]
+                }
+            ]
+        }
+    }
+
+**Response Body Structure**:
+............................
+
+- (dict) --
+    - ``status_code`` (int) --
+        An integer indicates the outcome status, including:
+            - ``200``: request processed successfully.
+            - ``400``: the request payload has JSON format errors.
+            - ``401``: the request payload missed certain fileds, required by :ref:`Payload specification <reat-time-payload-spec>`.
+            - ``402``: the request payload missed values on certain fileds.
```

Comment:
Can we be more specific about the difference between these two, e.g. with examples?